ext-src/browser.ts

```diff
@@ -48,9 +48,12 @@ export default class Browser extends EventEmitter {
       chromeArgs.push('--no-sandbox');
     }
 
+    let extensionSettings = vscode.workspace.getConfiguration('browser-preview');
+    let ignoreHTTPSErrors = extensionSettings.get<boolean>('ignoreHttpsErrors');
```

Comment:
How does this resolve if there's no settings or property isn't around?

Reply:
Currently it is an empty string if it is not set in settings.json, which works fine as it's treated the same by puppeteer as false. I can however change the default to false in package.json.
I could also set a default in the code or even make a check and set it to false if it's not a bool.